src/scalib_ext/scalib/src/mttest.rs

```diff
@@ -552,8 +552,8 @@ fn inner_prod_generic(
     for j in 0..ts[0].len() {
         for i in 0..prods.len() {
             let (t, c) = posi[i];
-            let (prod_old, mut prod_up) = prods.split_at_mut(i);
-            let (_, mut accs_up) = accs.split_at_mut(i);
+            let (prod_old, prod_up) = prods.split_at_mut(i);
```

Comment:
Don't think these changes belong in this PR, but looks harmless enough

Reply:
That's just fixing compiler warnings (in general, removing a `mut` in rust will never change the behavior of the code -- only possibly fail compilation).